pyfepdf.py

```diff
@@ -1541,7 +1541,7 @@ def ProcesarPlantilla(self, num_copias=3, lineas_max=36, qty_pos="izq"):
                     else:
                         barras = ""
 
-                    f.set("CodigoBarras", unicode(barras))
+                    f.set("CodigoBarras", barras)
```

Comment:
Convert to string, just in case the user sent a integer
```suggestion
                    f.set("CodigoBarras", str(barras))
```
Anyway, as in the previous comment, maybe it would be better to open a new PR with assorted changes with small bugfixes